.github/workflows/build-latest.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       xdr_ref: v22.0.0
       core_ref: v22.0.0
       horizon_ref: horizon-v22.0.1
-      stellar_rpc_ref: v22.1.0
+      stellar_rpc_ref: fb9f8c7c146485e0d4f7e4285e62e1678c89ce3d
```

Comment:
I don't think it's worth going through the hassle of releasing just for this. It will get updated to a stable tag in the next release.

Reply:
If it's not worth releasing rpc with the new name, whats the motivation for breaking out of the regular process, and releasing a special unreleased version into QuickStart now rather than updating it at the rpc next stable release?

Reply:
I would agree that quickstart should only have released versions referenced here.

Reply:
CI is broken in stellar-rpc because of this

Reply:
The CI that's broken, can you update it to point to this PR? This PR is published to: ``.

Reply:
OK, I will give that a try

Reply:
Can you take a look at https://github.com/stellar/system-test/pull/107 ?

Reply:
I have just changed the code to use rpc 22.1.1